### PR TITLE
Fix invisible brain image

### PR DESCRIPTION
**Pull request by toloudis**
Default LUT had 3 control points and our code was not handling it well.
In production viewer, this url would load and ultimately reset its luts to a generic fullrange LUT which made the image invisible.
With the changes, the volumes show up as expected.
https://allen-genetic-tools.s3.amazonaws.com/tissuecyte/1298553550/ome-zarr

The code change is just trying to make a reasonable guess at a min/max from the given LUT control points.

### CODE REVIEW

File: src/aics-image-viewer/shared/utils/controlPointsToLut.ts [hunk 1]
```diff
@@ -126,10 +126,23 @@ export function initializeLut(
...
   } else if (controlPoints.length === 2) {
     return [controlPoints[0].x, controlPoints[1].x];
+  } else if (controlPoints.length === 3) {
+    if (
+      controlPoints[0].opacity !== controlPoints[1].opacity &&
+      controlPoints[0].opacity !== controlPoints[2].opacity &&
+      controlPoints[1].opacity !== controlPoints[2].opacity
+    ) {
+      // if all 3 are unequal, assume a ramp from first to last
+      return [controlPoints[0].x, controlPoints[2].x];
```
Comment (ShrimpCryptid) [inline src/aics-image-viewer/shared/utils/controlPointsToLut.ts:140]: Could this be reduced to just a check for `controlPoints[0].opacity !== controlPoints[2].opacity`?

Comment (toloudis) [reply src/aics-image-viewer/shared/utils/controlPointsToLut.ts:140]: if you only check c0 != c2, then c0 could still equal c1, or c1 could be equal to c2.  You still need to make some decision on which are the two best control points to choose.

**Review by interim17 — APPROVED** [review]
LGTM

I keep reading the title of the PR as "invisible brain damage" and think I'm reviewing a punk album.

File: src/aics-image-viewer/shared/utils/controlPointsToLut.ts [hunk 2]
```diff
@@ -126,10 +126,23 @@ export function initializeLut(
 }
 
 export function controlPointsToRamp(controlPoints: ControlPoint[]): [number, number] {
-  if (controlPoints.length <= 1 || controlPoints.length === 3) {
+  if (controlPoints.length === 1) {
```
Comment (toloudis) [inline src/aics-image-viewer/shared/utils/controlPointsToLut.ts:129]:
```suggestion
  if (controlPoints.length <= 1) {
```

**Pull request merged**